appstudio-controller/controllers/appstudio.redhat.com/applicationsnapshotenvironmentbinding_controller_test.go

```diff
@@ -24,6 +24,7 @@ var _ = Describe("ApplicationSnapshotEnvironmentBinding Reconciler Tests", func(
 		var ctx context.Context
 		var request reconcile.Request
 		var binding *appstudiosharedv1.ApplicationSnapshotEnvironmentBinding
+		var bindingSecond *appstudiosharedv1.ApplicationSnapshotEnvironmentBinding
```

Comment:
Is it an intended behavior to duplicate import `*appstudiosharedv1.ApplicationSnapshotEnvironmentBinding` as binding and bindingsecond?

Reply:
All fields are same but I need different labels , so I have created a new one, I will update this as Jayendra mentioned in one of the comment